### PR TITLE
Modified Integration Test infra to add istio v1.7.3

### PR DESCRIPTION
**Pull request by anpanigr**
Modified Integration Test infra to support istio  v1.7.3

Note :  It requires minimum K8s version 1.16

Jenkin Parallel Run 
   https://build.weblogick8s.org:8443/job/wko-tmp/118/  ( all ISTIO tests passed )

### CODE REVIEW

**Review by jshum2479 — APPROVED** [review]
LGTM

File: integration-tests/src/test/resources/bash-scripts/install-istio.sh [hunk 1]
```diff
@@ -23,25 +26,23 @@ workdir=$2
 istiodir=${workdir}/istio-${version}
 echo "Installing Istio version [${version}] in location [${istiodir}]"
 
-helm repo add istio.io https://storage.googleapis.com/istio-release/releases/${version}/charts/
-
 kubectl delete namespace istio-system --ignore-not-found
-kubectl create namespace istio-system
+#kubectl create namespace istio-system
```
Comment (bhavaniravichandran) [inline integration-tests/src/test/resources/bash-scripts/install-istio.sh:30]: The ns is deleted but creating the ns is ocmmented out. If it is created by instrall, can you please add a comment saying that?

**Review by bhavaniravichandran — CHANGES_REQUESTED** [review]

File: integration-tests/src/test/resources/bash-scripts/install-istio.sh (hunk 1 above)
Comment (anpanigr) [reply integration-tests/src/test/resources/bash-scripts/install-istio.sh]: Added the comments 
>
istio installation will create the namespace 'istio-system
>

**Review by bhavaniravichandran — APPROVED** [review]
LGTM

**Review by mriccell — APPROVED** [review]

**Pull request merged**